homeassistant/components/hvac/zwave.py

```diff
@@ -204,20 +214,23 @@ def set_temperature(self, temperature):
 
     def set_fan_mode(self, fan):
         """Set new target fan mode."""
-        for value in self._node.get_values(class_id=0x44).values():
+        for value in self._node.get_values(
+                class_id=COMMAND_CLASS_THERMOSTAT_FAN_MODE).values():
             if value.command_class == 68 and value.index == 0:
                 value.data = bytes(fan, 'utf-8')
 
-    def set_operation_mode(self, operation_mode):
+    def set_operation(self, operation_mode):
```

Comment:
Services have changed name, and will be in 0.20 ( #2017 ) `set_operation_mode` is the correct name.
Same with `set_swing`